recipes/biodigest/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "biodigest" %}
+{% set version = "0.2.16" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 7618cedf40fedd6dd60aa2fbf8635fb4af0a2c0ae16c01ef9741d30729b6e8bc
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('biodigest', max_pin="x.x") }}
+
+requirements:
+  host:
+    - biothings_client ==0.2.6
+    - gseapy ==0.10.5
+    - numpy ==1.24.3
+    - pandas ==1.5.2
+    - pip
+    - psutil ==5.9.0
+    - pycairo >=1.21.0
+    - python
+    - requests ==2.28.2
+    - scipy ==1.8.0
+    - seaborn ==0.12.2
+  run:
+    - biothings_client ==0.2.6
+    - gseapy ==0.10.5
+    - graph-tool
+    - numpy ==1.24.3
+    - pandas ==1.5.2
+    - psutil ==5.9.0
+    - pycairo >=1.21.0
+    - python
+    - requests ==2.28.2
+    - scipy ==1.8.0
+    - seaborn ==0.12.2
+
+
+test:
+  imports:
+    - biodigest
+    - biodigest.evaluation
+    - biodigest.evaluation.d_utils
+    - biodigest.evaluation.mappers
+
+about:
+  home: "http://pypi.python.org/pypi/biodigest/"
+  license: LICENSE
+  license_family: OTHER
+  license_file: 
+  summary: "In silico Validation of Disease and Gene Sets, Clusterings or Subnetworks (DIGEST)"
+  doc_url: 
+  dev_url: 
+
+extra:
+  recipe-maintainers:
+    - your-github-id-here
```

Comment:
Could you add your github id here? otherwise this can be deleted

Reply:
I did, thanks.